Dart/resources/META-INF/plugin.xml

```diff
@@ -289,5 +290,22 @@
       <separator/>
     </group>
 
+    <action id="DartCallHierarchy.BaseOnThisFunction" text="Base on This Function"
+            class="com.jetbrains.lang.dart.ide.hierarchy.call.DartCallHierarchyBrowser$BaseOnThisFunctionAction"
```

Comment:
Since `DartCallHierarchyBrowser$BaseOnThisFunctionAction` is empty I think removing it and pointing here to `com.intellij.ide.hierarchy.CallHierarchyBrowserBase$BaseOnThisMethodAction` should work